pkg/healthcheck/healthcheck_output.go

```diff
@@ -23,6 +23,8 @@ const (
 	TableOutput = "table"
 	// WideOutput is used to specify the wide output format
 	WideOutput = "wide"
+	// ShortOutput is used to specify the short output format
+	ShortOutput = "short"
```

Comment:
Can be a follow-up since it's not really related, but it would be nice if we can use these constants in the `cli/cmd` package instead of defining them twice.